src/Yoti.Auth/Conversion.cs

```diff
@@ -31,8 +31,11 @@ public static byte[] Base64ToBytes(string base64)
         /// </summary>
         public static byte[] UrlSafeBase64ToBytes(string urlSafeBase64)
         {
+#if NETCOREAPP2_2
```

Comment:
Looks like 2.0 supports this https://docs.microsoft.com/en-us/dotnet/api/system.string.replace?view=netcore-2.0#System_String_Replace_System_String_System_String_System_StringComparison_ - should this be changed to `>= 2.0` or `2.1` to match _TargetFrameworks_ config

Reply:
Have changed this to have 2.0 & 2.1 in c140a77